packages/firestore/test/util/test_platform.ts

```diff
@@ -45,7 +45,8 @@ export class FakeWindow {
     });
     this.fakeIndexedDb =
       fakeIndexedDb ||
-      (typeof window !== 'undefined' && window.indexedDB) || null;
+      (typeof window !== 'undefined' && window.indexedDB) ||
+      null;
```

Comment:
This shows up in every one of my diffs... (after running prettier on all sources)

Reply:
neat!